game_eggs/minecraft/java/nanolimbo/egg-nano-limbo.json

```diff
@@ -0,0 +1,50 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PTERODACTYL PANEL - PTERODACTYL.IO",
+    "meta": {
+        "version": "PTDL_v1",
+        "update_url": null
+    },
+    "exported_at": "2021-11-24T20:42:37+01:00",
+    "name": "NanoLimbo",
+    "author": "mail@wuffy.eu",
+    "description": "This is lightweight minecraft limbo server, written on Java with Netty. The main goal of the project is maximum simplicity with a minimum number of sent and processed packets. This limbo is empty, there are no ability to set schematic building since this is not necessary. You can send useful information in chat or BossBar.\r\n\r\nNo plugins, no logs. The server is fully clear. It only able keep a lot of players while the main server is down.",
+    "features": null,
+    "images": [
+        "quay.io\/pterodactyl\/core:java"
+    ],
+    "file_denylist": [],
+    "startup": "java -Xms128M -Xmx{{SERVER_MEMORY}}M -jar {{SERVER_JARFILE}}",
+    "config": {
+        "files": "{\r\n    \"settings.yml\": {\r\n        \"parser\": \"yaml\",\r\n        \"find\": {\r\n                \"bind.ip\": \"0.0.0.0\",\r\n                \"bind.port\": \"{{server.build.default.port}}\"\r\n        }\r\n    }\r\n}",
+        "startup": "{\r\n    \"done\": \"Server started on\"\r\n}",
+        "logs": "{}",
+        "stop": "^C"
+    },
+    "scripts": {
+        "installation": {
+            "script": "#!\/bin\/bash\r\n# NanoLimbo Installation Script\r\n#\r\n# Server Files: \/mnt\/server\r\n\r\napt update\r\napt install -y curl git openssl bash bc apt-transport-https gnupg software-properties-common\r\n\r\nif [ -z \"${DL_PATH}\" ]; then\r\n    DL_PATH=`curl -Ls -o \/dev\/null -w %{url_effective} https:\/\/github.com\/Nan1t\/NanoLimbo\/releases\/latest`\r\n    DL_PATH=$(grep -oP '[0-9].[0-9].[0-9]' <<< \"$DL_PATH\")\r\n    DL_PATH=https:\/\/github.com\/Nan1t\/NanoLimbo\/releases\/download\/v$DL_PATH\/NanoLimbo-$DL_PATH-all.jar\r\nfi\r\n\r\ncd \/mnt\/server\r\n\r\nif [ ! -f \"settings.yml\" ]; then\r\n    echo \"Downloading config file...\"\r\n    curl -sSL -o settings.yml https:\/\/gist.githubusercontent.com\/NgLoader\/c1ec994cc426a2b0ce0b4245ebf9e412\/raw\/38e94ec5c31f987c25688b4bfd007967d983f187\/settings.yml\r\nfi\r\n\r\necho \"Downloading server file...\"\r\ncurl -sSL -o ${SERVER_JARFILE} ${DL_PATH}",
```

Comment:
We actually have an existing [Github release grabber script](https://github.com/parkervcp/eggs/blob/master/scripts/github_release_grabber.sh) that takes advantage of their API

Reply:
nice

Reply:
@Software-Noob How do I get the download link?

```java
export GITHUB_PACKAGE="Nan1t/NanoLimbo"
export MATCH="v"
curl -s https://raw.githubusercontent.com/parkervcp/eggs/master/scripts/github_release_grabber.sh | bash
```

Reply:
Copy the contents of the script into the install script of the egg itself. There's no reason to pass it to bash.

Reply:
Okay